flink-runtime/src/main/java/org/apache/flink/runtime/taskexecutor/FileType.java

```diff
@@ -22,6 +22,13 @@
  * Different file types to request from the {@link TaskExecutor}.
  */
 public enum FileType {
+	/**
+	 * the log file type for taskmanager
```

Comment:
this seems unrelated?

Reply:
Somewhat. But I believe that the extra effort splitting this change into a separate commit won't pay off. Hence, I would like to keep it as is.

Reply:
Per the [code-style guide](https://flink.apache.org/contributing/code-style-and-quality-pull-requests.html#separate-refactoring-cleanup-and-independent-changes):
> Pull Requests must put cleanup, refactoring, and core changes into separate commits.

Reply:
You are right Chesnay. I will refactor the changes into separate commits in order to follow the code style guide.